src/test/frontend/handler/editor_test.cljs

```diff
@@ -4,10 +4,17 @@
             [clojure.test :refer [deftest is testing are use-fixtures]]
             [datascript.core :as d]
             [frontend.test.helper :as test-helper :refer [load-test-files]]
+            [frontend.db.model :as model]
             [frontend.state :as state]
             [frontend.util.cursor :as cursor]))
 
-(use-fixtures :each test-helper/start-and-destroy-db)
+(use-fixtures :each {:before (fn []
```

Comment:
We should use the previous implementation as the new fixture is just a copy of this